web/sessions_controller_test.go

```diff
@@ -51,7 +51,7 @@ func TestSessionsController_create(t *testing.T) {
 				assert.Equal(t, 1, len(cookies))
 				decrypted, err := cltest.DecodeSessionCookie(cookies[0].Value)
 				require.NoError(t, err)
-				user, err := app.Store.AuthorizedUserWithSession(decrypted)
+				user, err := app.Store.AuthorizedUserWithSession(decrypted, app.Config.SessionTimeout.Duration)
```

Comment:
The config is already on the store, so it shouldn't need to be passed in. Is this for testability? Seems like we could cut the second param.